generic_config_updater/generic_updater.py

```diff
@@ -156,11 +142,6 @@ def checkpoint(self, checkpoint_name):
         self.logger.log_notice("Getting current config db.")
         json_content = self.config_wrapper.get_config_db_as_json()
 
-        # if current config are not valid, we might not be able to rollback to it. So fail early by not taking checkpoint at all.
-        self.logger.log_notice("Validating current config according to YANG models.")
```

Comment:
No need to fail while taking a checkpoint, we fail when applying the patch if the config is not valid or we can decide to use the non-strict mode and skip validation